prometheus/generic/setup-bind-exporter/README.md

```diff
@@ -0,0 +1,52 @@
+setup-node-exporter
+=========
+
+This role will instantiate a bind-exporter container on targeted hosts. The statistics page has to be enabled and available locally on targeted hosts.
```

Comment:
Can you provide a link to describe how to do this?

Reply:
This should be resolved now. Added link to the docs and provided configuration snippet in the readme file